### PR TITLE
Updated AMI references to whatever the latest ECS AMI is at the moment

### PR DESCRIPTION
**Pull request by jasiek**

### CODE REVIEW

**Comment by antonbabenko** [comment]
Would be great to remake this to use data source instead of hard-coded values for AMI IDs.

@jasiek @tfhartmann Will you guys be able to do this in the future?

**Comment by tfhartmann** [comment]
@antonbabenko yes! I absolutely agree! It's actually been on my to-do list to do!  For this exact reason.  I do want to make sure that the data source doesn't just return the most recent version of the AMI though, in case people are running applications that would be adversely effected by a bulk removal of all the instances.  Which is the behavior of this module currently. For the most part, it hans't been an issue since the containers get restarted quickly. But I've run into a couple of spots already where it we'd benefit for a less aggressive scaling policy.   
 @jasiek thanks for the PR, I'll merge it and cut a new release tag.

**Comment by jasiek** [comment]
I've added a commit that should compute the AMI image id dynamically, but need to test it (tomorrow).

**Comment by jasiek** [comment]
Tested and done!

File: main.tf [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+  most_recent = true
+
+  filter {
+    name = "owner-alias"
+    values = ["amazon"]
+  }
+
+  filter {
+    name = "name"
+    values = ["amzn-ami-*.*.g-amazon-ecs-optimized"]
```
Comment (antonbabenko) [inline main.tf]: In order to make it more flexible you can replace it with: `values = ["amzn-ami-${var.ami_version}.g-amazon-ecs-optimized"]` and define variable like this:
```
variable "ami_version" {
  default = "*"
}
```
and update example code in README.

This way users can provide `ami_version` to the module, if they don't want to have unexpected upgrades.

Comment (jasiek) [reply main.tf]: Good idea.

**Comment by jasiek** [comment]
Tested again, appears to be working.

**Comment by antonbabenko** [comment]
LGTM

**Comment by tfhartmann** [comment]
Looks good to me too!

**Pull request merged**